### PR TITLE
Escape special chars

### PR DESCRIPTION
**Pull request by Europium-152**
There are changes in two files:

create matrix.js
range to string.js

these changes allow for the escaping of user-defined characters

### CODE REVIEW

**Comment by Europium-152** [comment]
I wasn't, thanks for noticing. Now they are all being correctly escaped.

File: create matrix.js [hunk 1]
```diff
@@ -1,3 +1,26 @@
+// This function is called by method string.replace() to correctly escape LaTex characters
+function myReplacer(match){
```
Comment (caenrigen) [inline create matrix.js:2]: using "myStuff" naming seems a bit silly

File: create matrix.js [hunk 2]
```diff
@@ -1,3 +1,26 @@
...
+  
+  // These chars are escaped prepending a backslash
+ if (match=="&" || match=="%" || match=="$" || match=="#" || match=="_" || match =="{" || match =="}")
```
Comment (caenrigen) [inline create matrix.js:5]: what about using a dictionary like
`escapeCharsDict={"&":"\&",
"%":"\%"
}`
and retrieving a substitution with `escpaeCharsDict["&"]` ?

**Review by caenrigen — CHANGES_REQUESTED** [review]
Some code improve may be needed

**Pull request merged**